App/Lettuce/Sources/Lettuce/MessageCenter/Requests/messaging/message-attachments/DownloadAttchmentRequest.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+import HTTPTypes
+
+import Onion
+
+//https://developer.allegro.pl/documentation#operation/uploadAttachmentPUT
+public struct DownloadAttchmentRequest {
```

Comment:
Should conform to `UploadRequest` protocol :) You can check [WriteNewMessageRequest](https://github.com/sloik/seller/blob/main/App/Lettuce/Sources/Lettuce/MessageCenter/Requests/messaging/messages/WriteNewMessageRequest.swift) for inspiration with that :)